packages/python/build

```diff
@@ -18,6 +18,12 @@ rm -rf "$PKG_PATH/lib/python3.4/test"
 # mkpanda option to apply to any package.
 find "$PKG_PATH/lib/" ! -type d -name "*.a" -exec rm -f -- '{}' +
 
+# This works around the issue of Python's build not being able to hardlink these files.
+# This is due to the build being performed inside of the VM's shared folder.
+# https://www.virtualbox.org/ticket/818
+if [[ "$OS" == "darwin" ]]; then
+    cp -f "$PKG_PATH/bin/python3.4m" "$PKG_PATH/bin/python3.4"
```

Comment:
Strange, how did you hit that on OS X? I am unable to reproduce your error. Although it maybe be because I'm building with Docker Machine and I'm assuming that the hard link won't work. What I can do is make a copy to some temporary name and then swap the file instead. What do you think?